packages/cli-hydrogen/src/cli/commands/hydrogen/deploy.ts

```diff
@@ -24,6 +24,11 @@ export default class Deploy extends Command {
       env: 'SHOPIFY_HYDROGEN_FLAG_COMMIT_AUTHOR',
       description: 'Override the default Git commit author.',
     }),
+    pathToBuild: Flags.string({
+      hidden: true,
+      env: 'SHOPIFY_HYDROGEN_FLAG_PATH_TO_BUILD',
+      description: 'Skip build process and use provided value as build',
```

Comment:
@gfscott Could use some 👀 on wording here.

Reply:
Cool! Questions:
1. What's the default behavior if they _don't_ pass this flag?
2. The path they specify: is it intended to point to a fully built project that was created by some other build process? Or is it a different build command? Not clear to me from the current wording!

Reply:
> What's the default behavior if they don't pass this flag?

We build the project ourself either in the current working directory or in the one provided by the `path` argument.

> is it intended to point to a fully built project that was created by some other build process?

Yes

Reply:
Does this work?

```suggestion
      description: 'Skip the build process and deploy the built project at the specified path. You’ll need to manage your own project build step.',
```